### ISSUE TITLE
Ability to set additional proxy info like name and voting philosophy 

### ISSUE THREAD

**Issue by n8d** [fioprotocol/fio#142]
It would be great to have an API call for proxies to set optional additional information about themselves such as name, website, and voting philosophy. This is typically implemented on EOSIO chains on the `regproxyinfo` account with this contract:

https://github.com/AlohaEOS/eos-proxyinfo

Perhaps for FIO it's a new API endpoint such as `/register_proxy_info` and it could use the same `proxies` table but on the `eosio` account. Using the same table structure will make it easier for existing EOSIO tools and wallets to integrate.

**Comment by lukestokes** [comment]
Yes, I think this would be a great idea. @n8d you interested in writing up a <a href="https://github.com/fioprotocol/fips">FIP</a> for it? It would also have to have fees associated to the call.
